README.md

```diff
@@ -156,7 +154,7 @@ Options:
   --help                Show this message and exit.
 
 Commands:
-  clean      Remove subfolders/files not matching .nbautoconvert...
+  clean      (EXPERIMENTAL) Remove subfolders/files not matching...
```

Comment:
What's the `...` for here?

Reply:
That's just what `typer`/`click` do when the docstring is too long and they truncate it.

Reply:
Ah right forgot this section is copied and pasted in